pkg/roll/farm.go

```diff
@@ -134,6 +135,12 @@ START_LOOP:
 					rlLogger.WithError(err).Errorln("Could not read new RC")
 					continue
 				}
+
+				if rcField == nil {
```

Comment:
I think I would have preferred modeling this after the `os` package. 

```
if rcs.IsNotExist(err) {
 ...
}
else if err != nil {
 ... 
}
```

Adding a new pointer makes it possible to test for nil in places, but it also makes it hard to use the zero value of the struct safely. Not a serious enough problem to block this PR, just expressing that the `return nil, nil` case from `rcs.Get` deserves something a bit more structurally meaningful.